### PR TITLE
Cw/text onno

### PR DESCRIPTION
**Pull request by chriswynn**

### CODE REVIEW

File: src/components/Text/index.tsx [hunk 1]
```diff
@@ -1,44 +1,47 @@
...
+      : props.theme.fonts.main,
+    marginBottom: get(["spaces", props.fontSize], props.theme),
+    lineHeight:
+      props.fontSize === 4
+        ? props.theme.lineHeights[2]
+        : props.fontSize === 0
+        ? props.theme.lineHeights[0]
+        : props.theme.lineHeights[1],
+    opacity: props.dim ? 0.5 : 1,
+  }),
```
Comment (sethkrasnianski) [inline src/components/Text/index.tsx]: What happens if a user adds a `css` or `style` prop to one of these onno-ified components?

Comment (chriswynn) [reply src/components/Text/index.tsx]: So the point of using onno is so you don't need to use the css prop. You would define styles with props like so `<Text marginBottom={3} marginTop={2} color="red">....</Text>`. If you do need to use the css prop the styles in the css prop will overwrite the preexisting ones.

File: src/components/AutoGrid/index.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const AutoGrid: React.FC<Props> = ({ gap = 1, columns = [1, 5], ...props }) => (
       mq({
         display: "grid",
         width: "100%",
-        gridGap: gap && theme.spacing[gap],
```
Comment (sethkrasnianski) [inline src/components/AutoGrid/index.tsx]: What's the rationale behind this name change?

Comment (chriswynn) [reply src/components/AutoGrid/index.tsx]: The default key to use the array values for space in onno is spacing. You can modify it and use your own name, but we decided to use the default naming convention onno uses.

**Pull request merged**